### PR TITLE
Add billing events

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/cloud/issues/442

#### Description
Add billing event to audit logging: update account, create/delete/update credit card

#### Dependency
https://github.com/gravitational/teleport/pull/6037
https://github.com/gravitational/teleport.e/pull/239

### CODE REVIEW

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 1]
```diff
@@ -306,6 +306,22 @@ export const formatters: Formatters = {
     format: ({ user, mfa_device_name, mfa_device_type }) =>
       `User [${user}] deleted ${mfa_device_type} device [${mfa_device_name}]`,
   },
+  [CodeEnum.BILLING_CARD_CREATE]: {
+    desc: 'Billing Credit Card Created',
+    format: ({ user }) => `A billing credit card has been created by ${user}`,
```
Comment (alex-kovoy) [inline packages/teleport/src/services/audit/makeEvent.ts:311]: @kimlisa 
Let's be consistent with other events 
-> `User [${user}] has updated a credit card`

Comment (alex-kovoy) [reply packages/teleport/src/services/audit/makeEvent.ts:311]: btw, feel free to pick an icon to use for billing events.

Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts]: revised
image

Comment (alex-kovoy) [reply packages/teleport/src/services/audit/makeEvent.ts]: Lets change the event description to:

"Credit Card Created"
"Credit Card Updated"
"Credit Card Deleted"
"Billing Information Updated"

Comment (alex-kovoy) [reply packages/teleport/src/services/audit/makeEvent.ts]: Billing Account -> Billing Information

Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts]: revised
image

**Review by alex-kovoy — APPROVED** [review]

**Pull request merged**